engine/runtime/src/test/java/org/enso/interpreter/test/tck/EnsoTckErrorTypeTest.java

```diff
@@ -0,0 +1,7 @@
+package org.enso.interpreter.test.tck;
+
+public class EnsoTckErrorTypeTest extends com.oracle.truffle.tck.tests.ErrorTypeTest {
+  public EnsoTckErrorTypeTest(com.oracle.truffle.tck.tests.TestRun r) {
+    super(r);
+  }
+}
```

Comment:
Are these tests supposed to be implemented later or just the class declaration is enough?

Reply:
The JUnit framework scans the not only this class, but also its super class for various method declarations. If we used `maven-surefire-plugin` then the recommended configuration would be [one to scan also classes in other JARs](https://github.com/graalvm/simplelanguage/blob/master/tck/pom.xml#L73).

However I don't think `sbt` offers similar capability, so I manually scanned thru `*Test` classes in `org.graalvm.truffle:truffle-tck-tests` and created a subclass for each of them. Brutal force, but probably good enough to start with. One needs to be careful when _upgrading to new version of GraalVM_ and list other modules as well.